R/do_retention_cohort.R

```diff
@@ -1,5 +1,5 @@
 #' Run timeseries cohort analysis
-#' @param df - Data frame to run bayes ab test
+#' @param df - Data frame to run do_retention_cohort
```

Comment:
Thank you for pointing this out.
The function name is actually do_cohort.

Reply:
Correct. Typos all over :)

Reply:
Thank you for correcting it!